### PR TITLE
security fix for path-parse

### PR DESCRIPTION
**Pull request by isc-rsingh**
This PR fixes dependabot alerts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1148,7 +1149,6 @@
     "vscode-debugadapter": "^1.41.0",
     "vscode-debugprotocol": "^1.41.0",
     "vscode-extension-telemetry": "^0.1.6",
-    "ws": "^7.4.6",
```
Comment (daimor) [inline package.json:1151]: Why did you remove 'we'?

Comment (isc-rsingh) [reply package.json:1151]: It's still there. Down lower.
